pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -45,161 +45,195 @@ msgid ""
 "\n"
 "The {event.name} organisers\n"
```

Comment:
"Das {event.name} Organisationsteam"

Reply:
Accendental comment, this one is the english one.